File: examples/python/list_phone_numbers.py
"""
List all phone numbers attached to your Inkbox account.

Usage:
    INKBOX_API_KEY=sk-... python list_phone_numbers.py
"""

import os
from inkbox.phone import InkboxPhone

client = InkboxPhone(api_key=os.environ["INKBOX_API_KEY"])

numbers = client.numbers.list()

for n in numbers:
    print(f"{n.number}  type={n.type}  status={n.status}")

File: examples/typescript/list-phone-numbers.ts
/**
 * List all phone numbers attached to your Inkbox account.
 *
 * Usage:
 *   INKBOX_API_KEY=sk-... npx ts-node list-phone-numbers.ts
 */

import { InkboxPhone } from "../../typescript/src/phone/index.js";

const client = new InkboxPhone({ apiKey: process.env.INKBOX_API_KEY! });

const numbers = await client.numbers.list();

for (const n of numbers) {
  console.log(`${n.number}  type=${n.type}  status=${n.status}`);
}
